packages/chevrotain/src/parse/cst/cst_visitor.ts

```diff
@@ -147,5 +136,5 @@ export function validateMissingCstMethods(
     }
   )
 
-  return compact<IVisitorDefinitionError>(errors)
```

Comment:
I noticed that this `compact` isn't covered by the test suite, or perhaps it's unnecessary. Using just `return errors`, the tests still pass.